### ISSUE TITLE
Specific chunks appear invisible 

### ISSUE THREAD

**Issue by D4rkMn** [TheCBProject/CodeChickenLib#464]
I have a modded server with some friends, and one of them reported that some specific chunks from our base have seemingly disappeared:

![image](https://github.com/user-attachments/assets/03508b49-7bfd-44d1-93b9-c394d51e020a)

After digging into the issue, i found out that removing CodeChickenLib (and other mods dependent on it) and loading into the world for some reason fixed the issue. The mods i removed were the following:

![image](https://github.com/user-attachments/assets/205d4f66-04e9-4c00-9e7d-a910773a897d)

Then i tried putting those mods back after the chunks returned to see if the chunk would be gone again. Turns out the chunk was still there. So the solution ended up being removing these mods, loading the world (which would restore the chunks), and then put the mods back in. So here i am just informing.

This issue was hard to trace to these mods as there were almost no logs that informed me of this mod being the issue. I only found a single line mentioning it and decided to give it a go, and it worked.

Minecraft version: 1.18.2

Forge version: 40.2.21

The rest of the mods in my client are the following:
[https://pastebin.com/B1XezN7V](https://pastebin.com/B1XezN7V)

**Comment by FoxMcloud5655** [comment]
A very weird issue to be sure, but I doubt 1.18.2 is being supported anymore...  I could be wrong, but that's how it is for Draconic Evolution, so it's probably the same for CodeChickenLib.

One question though, if you still have a backup of the corrupted world, did you try CodeChickenLib by itself while removing the other mods in that list?

**Comment by D4rkMn** [comment]
In the screenshot you can see there are about 6 chunks missing. While i was trying to find a solution i tried removing embeddium and dependents to see if that would do anything. That ended up restoring about 5 chunks, but one specific chunk was still missing. So i kept trying to remove other optimization mods (specifically mods that target rendering side) but that specific chunk was still missing. Then i tried removing CoreChickenLib and those other mods, and the chunk returned. So in the end none of these optimization / rendering mods being removed helped, only when removing CoreChickenLib.

**Comment by FoxMcloud5655** [comment]
Right, I get that, but did you try to remove (or if you have that backup, can you try now to remove) all the mods that depended on CodeChickenLib while keeping CodeChickenLib to see if CodeChickenLib was the one actually at fault here?

**Comment by D4rkMn** [comment]
Yes, thats what i did. The mods i removed are the ones in the second picture, all of which require CodeChickenLib as dependency to run. When removing these mods, the chunk was restored. Maybe is the fault of one of these other mods, like ProjectRed or another one of those mods, but once again it seems to work when removing these

**Comment by FoxMcloud5655** [comment]
Alright, so to be absolutely 100% clear, you kept CodeChickenLib installed as a mod, removed the mods that depended on CodeChickenLib without uninstalling CodeChickenLib, then booted up the pack and it was fixed...  Correct?
